src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -88,6 +101,8 @@ public String getConfirmation(@PathVariable String companyNumber,
         try {
             PayableLateFilingPenalty payablePenalty = payablePenaltyService
```

Comment:
I am thinking do we still need this payablePenalty or not, because we will be using lateFilingPenalty in stead.

Reply:
Still used in setUpPaymentDisplay to verify if .getPayment() is not null